tests/integration/tests.mk

```diff
@@ -48,7 +48,7 @@ test.integration.all: test.integration test.integration.kube test.integration.ra
 
 # Generate integration test targets for kubernetes environment.
 test.integration.%.kube:
-	$(GO) test -p 1 ${T} ./tests/integration/$*/... ${_INTEGRATION_TEST_WORKDIR_FLAG} ${_INTEGRATION_TEST_CIMODE_FLAG} -timeout 30m \
+	$(GO) test -p 1 ${T} ./tests/integration/$*/... ${_INTEGRATION_TEST_WORKDIR_FLAG} ${_INTEGRATION_TEST_CIMODE_FLAG} -timeout 60m \
```

Comment:
do you know why the test is taking so long?

Reply:
It was because of redis install.. I did some refactoring so that redis is installed only once for all redis tests.. removing this to see if tests pass in prow